pages/index.tsx

```diff
@@ -112,8 +113,15 @@ export default Frontpage
 export const getServerSideProps: GetServerSideProps = async () => {
   const USE_IMPROVED_SEARCH = process.env.USE_IMPROVED_SEARCH === "true"
 
+  const indexName = process.env.ALGOLIA_SEARCH_INDEX ?? ""
+
+  if (indexName === "") {
+    console.log("Missing the Algolia search index name")
```

Comment:
we should probably handle this case in the UI somehow. can probably do that in a later pr